Test/function_test.py

```diff
@@ -342,6 +342,55 @@
 0.55
 >>> cm.aickin_alpha(max_iter=None)
 'None'
+>>> cm.positions
+>>> POS = cm.position()
+>>> POS == cm.positions
+True
+>>> POS[0]['TP']
+[1, 4, 9]
+>>> POS[0]['TN']
+[2, 3, 5, 6, 8, 10, 11]
+>>> POS[0]['FP']
+[0, 7]
+>>> POS[0]['FN']
+[]
+>>> POS[1]['TP']
+[6]
+>>> POS[1]['TN']
+[0, 1, 2, 4, 7, 8, 9, 11]
+>>> POS[1]['FP']
+[3]
+>>> POS[1]['FN']
+[5, 10]
+>>> POS[2]['TP']
+[2, 8, 11]
+>>> POS[2]['TN']
+[1, 4, 6, 9]
+>>> POS[2]['FP']
+[5, 10]
+>>> POS[2]['FN']
+[0, 3, 7]
+>>> y_actu = [0, 0, 1, 1, 0]
+>>> y_pred = [0, 1, 1, 0, 0]
+>>> cm2 = ConfusionMatrix(actual_vector=y_actu, predict_vector=y_pred)
+>>> POS = cm.position()
+>>> POS[0]['TP']
+[1, 4, 9]
+>>> POS[0]['TN']
+[2, 3, 5, 6, 8, 10, 11]
+>>> POS[0]['FP']
+[0, 7]
+>>> POS[0]['FN']
+[]
+>>> POS[1]['TP']
+[6]
+>>> POS[1]['TN']
+[0, 1, 2, 4, 7, 8, 9, 11]
+>>> POS[1]['FP']
+[3]
+>>> POS[1]['FN']
+[5, 10]
+>>> cm2.relabel({0:'L1',1:'L2'})
```

Comment:
Please read relabeled classes after this line, for example :
```pycon
>>> POS = cm.position()
>>> POS["L1"]["TP"]
```
You should update `self.positions` in `relabel` method

Reply:
Ok
I'll handle it.